lib/ffprober.rb

```diff
@@ -5,6 +5,7 @@
 require_relative 'ffprober/stream'
 require_relative 'ffprober/audio_stream'
 require_relative 'ffprober/video_stream'
+require_relative 'ffprober/subtitle_stream'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.